lisa/tools/lsvmbus.py

```diff
@@ -128,7 +128,9 @@ def install(self) -> bool:
             package_name = "linux-cloud-tools-common"
         if package_name:
             linux_os.install_packages(package_name)
-        else:
+        if not self._check_exists():
+            if package_name:
+                self._log.info(f"{package_name} not found or failed to install.")
```

Comment:
It can explain clearer, like below.

`"failed to install lsvmbus by package '{package_name}', trying to install by downloading."`

The reasons are,

1. To explain the next action is taken by our code. So that people know it's an information, no need to take extra action. Especially, we're saying 'failed'
2. add the "lsvmbus", since the log may be combined with other information log, it's better to explain the whole things in one line.
3. Add "package" and quote to the package_name. The package_name may not explain itself, we need to let users know, it's a package name.

Reply:
is it fixed?